generic3g/specs/StateItemSpec.F90

```diff
@@ -205,6 +208,15 @@ function I_get_aspect_priorities(src_spec, dst_spec) result(aspect_order)
          class(StateItemSpec), intent(in) :: dst_spec
       end function I_get_aspect_priorities
 
+!#      function I_make_extension(this, aspect_name, aspect, rc) result(new_spec)
```

Comment:
Why do we have a function that is commented out tottaly?

Reply:
Ah - this is another one of those temporary expediencies.  Eventually we want an abstract interface here, but that would force me to first provide a stub function in all the concrete subclasses.    Instead I provide a stub function in the base class (further down).  Again the intent is to clean up later on when each subclass has been properly updated.  The trick will be to remember and go back to fix-up the base class.

I'll try to be better about adding TODO's but ... I've never actually found myself going back to find "TODO" in an implementation.  The cleanup tends to happen more organically than that I've found.